models/constants.js

```diff
@@ -197,6 +197,12 @@ module.exports = {
      */
     CAMP_MEMBER_STATUS: ['approved', 'pending', 'pending_mgr', 'rejected', 'approved_mgr', 'supplier', 'deleted'],
 
-    EVENT_GATE_STATUS: get_enum(gate_status)
+    EVENT_GATE_STATUS: get_enum(gate_status),
+
+    // This is a list of URLs the login process is allowed to redirect to.
+    // This is to make sure users are not sent spark links via e.g. email by a malicious 3rd party
+    // and are redirected to the senders desired location, e.g. can be used for phishing.
+    // If we redirect to a new URL from login, we will need to add it here.
+    LOGIN_REDIRECT_URL_WHITELIST: ['/', '/admin'],
```

Comment:
I think this list should contain a whitelist of domain names (paths are easy to fake)
on production this list would be `["volunteers.spark.midburn.org","spark.midburn.org","dreams.midburn.org"]`
you can have it coming from environment variable and set for each environment

Reply:
This is just a redirect from login, i.e. the r QS parameter.
If dreams, volunteers, etc. use the same login, then sure, we could do that.